projects/fourier/lib/fourier/commands/bundle.rb

```diff
@@ -13,10 +13,39 @@ class Bundle < Base
         required: false,
         aliases: :p,
       )
+      option(
+        :xcode_version,
+        desc: %(
+          The version of Xcode to use to build tuist.
+          Can be either a version number (e.g. `--xcode_version="13.2.1"`)
+          or a file path pointing to the Xcode app bundle to use. (e.g. `--xcode_version="/path/to/Xcode.app"`)
+          Defaults to the currently selected Xcode on the system.
+        ),
```

Comment:
I wanted to find a way to parameterize this description, since it's basically the same for all the methods, but it wasn't working for me. Maybe you can't access methods in an initializer like this? Anyone know how I might accomplish that?